editor/import/resource_importer_scene.cpp

```diff
@@ -1408,144 +1414,138 @@ Ref<Animation> ResourceImporterScene::_save_animation_to_file(Ref<Animation> ani
 	return anim;
 }
 
-void ResourceImporterScene::_create_clips(AnimationPlayer *anim, const Array &p_clips, bool p_bake_all) {
-	if (!anim->has_animation("default")) {
-		ERR_FAIL_COND_MSG(p_clips.size() > 0, "To create clips, animations must be named \"default\".");
-		return;
-	}
+void ResourceImporterScene::_create_slices(AnimationPlayer *ap, Ref<Animation> anim, const Array &p_slices, bool p_bake_all) {
+	Ref<AnimationLibrary> al = ap->get_animation_library(ap->find_animation_library(anim));
 
-	Ref<Animation> default_anim = anim->get_animation("default");
-	Ref<AnimationLibrary> al = anim->get_animation_library(anim->find_animation(default_anim));
-
-	for (int i = 0; i < p_clips.size(); i += 7) {
-		String name = p_clips[i];
-		float from = p_clips[i + 1];
-		float to = p_clips[i + 2];
-		Animation::LoopMode loop_mode = static_cast<Animation::LoopMode>((int)p_clips[i + 3]);
-		bool save_to_file = p_clips[i + 4];
-		String save_to_path = p_clips[i + 5];
-		bool keep_current = p_clips[i + 6];
+	for (int i = 0; i < p_slices.size(); i += 7) {
```

Comment:
What does 7 mean, can you make it a named constant and the same with the ones below?

Reply:
I didn't touch most of the slicing logic, I just updated the variable names. I see it's 7 because each slice contains 7 import options. But it was like that before. I can refactor it once I get some more free time. Also I would like to see if there are more comments on this PR.